test/ec_internal_test.c

```diff
@@ -156,6 +156,54 @@ static int field_tests_ecp_mont(void)
 }
 
 #ifndef OPENSSL_NO_EC2M
+/* Test that decoding of invalid GF2m field parameters fails. */
+static int ec2m_field_sanity(void)
+{
+    int ret = 0;
+    BN_CTX *ctx = BN_CTX_new();
+    BIGNUM *p, *a, *b;
+    EC_GROUP *group1 = NULL, *group2 = NULL, *group3 = NULL;
+
+    TEST_info("Testing GF2m hardening\n");
+
+    BN_CTX_start(ctx);
+    if (!TEST_ptr(p = BN_CTX_get(ctx))
+        || !TEST_ptr(a = BN_CTX_get(ctx))
+        || !TEST_ptr(b = BN_CTX_get(ctx)))
```

Comment:
Strictly speaking it is only necessary to test the "last" `BN_CTX_get()` call (if one fails all subsequent `BT_CTX_get()` calls for the same ctx are guaranteed to return NULL by the API contract.

But ok, doesn't really matter.